lib/grack.rb

```diff
@@ -1,5 +1,10 @@
 require "grack/bundle"
-
+require 'zlib'
+require 'rack/request'
+require 'rack/response'
+require 'rack/utils'
+require 'time'
+require 'grack/git'
```

Comment:
If the convention is to have requires for that file be in that file I'll change that. I moved it so that each file isn't cluttered with duplicate requires.